packages/node_modules/@ciscospark/react-component-title-bar/src/index.js

```diff
@@ -4,13 +4,13 @@ import classNames from 'classnames';
 
 import styles from './styles.css';
 
-import Avatar from '@ciscospark/react-component-avatar';
+import PresenceAvatar from '@ciscospark/react-container-presence-avatar';
 
-export default function TitleBar({children, image, name}) {
+export default function TitleBar({avatarId, children, name}) {
   return (
     <div className={classNames(`ciscospark-title-bar`, styles.titleBar)}>
       <div className={classNames(`ciscospark-avatar-container`, styles.avatarContainer)}>
-        <Avatar image={image} name={name} />
+        <PresenceAvatar avatarId={avatarId} iconSize={`30px`} name={name} />
```

Comment:
Title bar avatars could also be space avatars right? Should there be a check here to make sure we pick the most efficient avatar component?

Reply:
Space avatars get their avatar urls from the same store users do, it would just fail to find presence for the space id.